hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/AddUpdateRolloutWindowLayout.java

```diff
@@ -868,6 +873,18 @@ private int getGroupSize() {
         return (int) Math.ceil((double) totalTargetsCount / Double.parseDouble(noOfGroups.getValue()));
     }
 
+    class TargetExistsValidator implements Validator {
```

Comment:
I would call the class different, e.g. TotalTargetsCountExistsValidator

Reply:
I prefer TargetExistsValidator